stats/dashboard.py

```diff
@@ -243,13 +243,18 @@ def valid_value(value_element):
 
 
 def valid_coords(x):
```

Comment:
Could we make life easier for ourselves in future by documenting this function to our better practices...

Suggest adding a docstring here